crates/router/tests/connectors/bambora.rs

```diff
@@ -0,0 +1,438 @@
+use masking::Secret;
+use router::types::{self, api, storage::enums};
+
+use crate::{
+    connector_auth,
+    utils::{self, ConnectorActions},
+};
+
+#[derive(Clone, Copy)]
+struct BamboraTest;
+impl ConnectorActions for BamboraTest {}
+impl utils::Connector for BamboraTest {
+    fn get_data(&self) -> types::api::ConnectorData {
+        use router::connector::Bambora;
+        types::api::ConnectorData {
+            connector: Box::new(&Bambora),
+            connector_name: types::Connector::Bambora,
+            get_token: types::api::GetToken::Connector,
+        }
+    }
+
+    fn get_auth_token(&self) -> types::ConnectorAuthType {
+        types::ConnectorAuthType::from(
+            connector_auth::ConnectorAuthentication::new()
+                .bambora
+                .expect("Missing connector authentication configuration"),
+        )
+    }
+
+    fn get_name(&self) -> String {
+        "bambora".to_string()
+    }
+}
+
+static CONNECTOR: BamboraTest = BamboraTest {};
+
+// Cards Positive Tests
+// Creates a payment using the manual capture flow (Non 3DS).
+#[actix_web::test]
```

Comment:
make the tests working for all the applicable scenarios, it is mandatory to have the tests working
please provide the screenshot for the same in the PR description